core/cap-0046-01.md

```diff
@@ -341,7 +406,7 @@ Benefits:
     - Easier to pass data from one contract to another.
     - Easier to use same data model from different source languages.
 
-It is especially important to note that the (enlarged) attack and maintenance surfaces on the host are costs borne by stellar-core developers, while the (diminished) attack and maintenance surfaces are benefits that accrue to smart contract developers. We believe this is a desirable balance of costs and benefits.
+It is especially important to note that the (enlarged) attack and maintenance surfaces on the host are costs borne by Soroban's developers, while the (diminished) attack and maintenance surfaces are benefits that accrue to smart contract developers. We believe this is a desirable balance of costs and benefits, as contract developers are likely to significantly outnumber Soroban developers.
 
 ### Rationale for value and object type repertoires
```

Comment:
I think in this section we should mention the choice of numbers repertoire, specifically:
- the choice of {u,i}128 and {u,i}256 over arbitrary precision big number
- the lack of floating point or fixed-point number types

Reply:
done